contracts/tokenWhitelist.sol

```diff
@@ -241,9 +241,15 @@ contract TokenWhitelist is ENSResolvable, Controllable, Transferrable {
     /// @notice Toggles whether or not a token is redeemable or not.
     function setTokenRedeemable(address _token, bool _redeemable) external onlyAdmin {
         // Require that the token exists.
-        require(_tokenInfoMap[_token].available, "token is not available");
-
-        // this sets the redeemable flag to the value passed in
+        require(_tokenInfoMap[_token].available, "redeemable: token not available");
+        // If it does not change the current state i.e. _tokenInfoMap[_token].redeemable == _redeemable, revert!
+        require(_tokenInfoMap[_token].redeemable != _redeemable, "redeemable: no state change");
```

Comment:
Do we want to fail here or just skip the assignment/counter logic?

Reply:
The reason I made it fail is that it it the pattern that we follow and it gives a warning to the sender that he is doing something wrong. Otherwise, the transaction will seem like everything was ok when there is the possibility that just the wrong argument was passed.